app/sequencescape/sequencescape/api/v2.rb

```diff
@@ -48,6 +48,10 @@ def self.plate_with_wells(uuid)
     Plate.includes('wells').find(uuid: uuid).first
   end
 
+  def self.tube_for_presenter(uuid)
+    Tube.includes('receptacle.aliquots.request.request_type').find(uuid: uuid).first
+  end
```

Comment:
Is this necessary? Seems like the 'includes' is taken care of by the 'DEFAULT_INCLUDES' on V2 Tube.

Also, where is it used?

Reply:
ah no, that can go, was used in an earlier iteration